Source/PINDiskCache.m

```diff
@@ -489,64 +489,83 @@ - (BOOL)_locked_createCacheDirectory
     return created;
 }
 
++ (NSArray *)resourceKeys {
+    static NSArray *resourceKeys = nil;
+    static dispatch_once_t predicate;
+
+    dispatch_once(&predicate, ^{
+        resourceKeys = @[ NSURLCreationDateKey, NSURLContentModificationDateKey, NSURLTotalFileAllocatedSizeKey ];
+    });
+
+    return resourceKeys;
+}
+
+- (void)_locked_initializeDiskPropertiesForFile:(NSURL *)fileURL fileKey:(NSString *)fileKey byteCount:(NSUInteger *)pByteCount {
```

Comment:
Would it be better to just return byteCount?

Reply:
Probably, although it's a little less obvious what it is